glide-core/redis-rs/redis/src/cluster_routing.rs

```diff
@@ -287,6 +384,27 @@ fn get_route(is_readonly: bool, key: &[u8]) -> Route {
     }
 }
 
+/// Represents the pattern of argument structures in multi-slot commands,
+/// defining how the arguments are organized in the command.
+#[derive(Debug, Clone, PartialEq)]
+pub enum MultiSlotArgPattern {
```

Comment:
So the idea is to cover all options here? e.g. if I look at `HSET`, which has `HSET KEY field value [field value..]` are we going to create a new enumerator here?

Reply:
HSET has a single key, so it wont be a multi-slot command. But, in general, yes - if there is going to be a command with a new pattern that we can support it in a multi-slot option, we will add it here